Onboarding/menu-non-personalised.md

```diff
@@ -0,0 +1,687 @@
+end
+
+card FamilyPlanning, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card HealthWorkers, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card MainMenu, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+```
+
+## Data Settings
+
+```stack
+card DataSettings, then: DisplayDataSettings do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_data_settings"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  loading_message = substitute(message.message, "{@username}", "@contact.name")
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+card DisplayDataSettings, then: DisplayDataSettingsError do
+  buttons(
+    AllData: "@button_labels[0]",
+    TextAndImages: "@button_labels[1]",
+    TextOnly: "@button_labels[2]"
+  ) do
+    text("@loading_message")
+  end
+end
+
+card DisplayDataSettingsError, then: DisplayDataSettingsError do
+  buttons(
+    AllData: "@button_labels[0]",
+    TextAndImages: "@button_labels[1]",
+    TextOnly: "@button_labels[2]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+card AllData, then: DataPreferencesConfirmation do
+  log("@preference")
+  data_preference = "all"
+  update_contact(data_preference: "all")
+end
+
+card TextAndImages, then: DataPreferencesConfirmation do
+  log("@preference")
+  data_preference = "text and images"
+  update_contact(data_preference: "text and images")
+end
+
+card TextOnly, then: DataPreferencesConfirmation do
+  log("@preference")
+  data_preference = "text only"
+  update_contact(data_preference: "text only")
+end
+
+```
+
+## Data Preferences Confirm
+
+```stack
+card DataPreferencesConfirmation, then: DisplayDataPreferencesConfirmation do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_data_preferences_confirmation"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+
+  loading_message =
+    substitute(
+      message.message,
+      "{Text only/ Text & images/ Text, images, audio & video}",
```

Comment:
Another variable name that needs replacing in the sheet. Maybe {data_preference}?